python-base/ubuntu22.04-python3.11-nginx-node18/config.yaml

```
@@ -0,0 +1,2 @@
+tags:
+  - ubuntu22-python3.11-nginx-node18
```

I'd prefer if it was `ubuntu22.04` instead of `ubuntu22`
```suggestion
  - ubuntu22.04-python3.11-nginx-node18
```